barcode_utils.py

```diff
@@ -0,0 +1,166 @@
+from barcode import *
+
+import base64
+import struct
+import skip32
+import code128
+import os
+
+
```

Comment:
I'll omit my questions about things like why we're referring to ``self`` in a non-instance method, since the comment already notes that this is broken.